client/elements/menus/more-par-menu.html

```diff
@@ -16,55 +17,100 @@
         color: var(--disabled-text-color);
       }
 
-      paper-item {
+      .button-text {
         color: inherit;
       }
+
+      .button-text:hover {
+        cursor: pointer;
+        background-color: var(--paper-grey-200);
+      }
     </style>
 
-    <paper-item on-tap="_copyLink">
+    <iron-ajax id="parallelsAjax"
```

Comment:
I use underscores instead of camelCase in my ID names, we can change to a different convention but let's stay consistent.